src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -383,7 +383,7 @@ private bool LaunchHost(TestProcessStartInfo testHostStartInfo, CancellationToke
                 }
                 else
                 {
-                    int processId = this.customTestHostLauncher.LaunchTestHost(testHostStartInfo);
+                    int processId = this.customTestHostLauncher.LaunchTestHost(testHostStartInfo, cancellationToken);
```

Comment:
We are breaking the API here. Is customTestHostLauncher used by others also? Interface ITesHostLauncher is in ObjectModel.
If its used by others, we need to introduce something called **IsCancellable** in ITestHostLauncher which will be used here to switch between overriden calls of customTestHostLauncher.LaunchTestHost.

Reply:
will it be used in UWP sceanrio ..?

---
In reply to: [181616862](https://github.com/Microsoft/vstest/pull/1543#discussion_r181616862) [](ancestors = 181616862)

Reply:
I confirmed from Mayank. It will not be breaking change.